starters/gatsby-starter-plugin/README.md

```diff
@@ -42,14 +42,14 @@ With `my-gatsby-site` being your Gatsby site, and `my-plugin` being your plugin.
 
 2. Include the plugin in a Gatsby site
 
-Inside of the `gatsby-config` file of your site (in this case, `my-gatsby-site`), include the plugin in the `plugins` array:
+Inside of the `gatsby-config.js` file of your site (in this case, `my-gatsby-site`), include the plugin in the `plugins` array:
 
-```
+```javascript:title=gatsby-config.js
```

Comment:
The title functionality does not work in READMEs and is only available on .org.

Reply:
i assumed that they maybe later also shared the README at the https://gatsbyjs.org/starters/gatsbyjs/gatsby-starter-plugin/ point like other starters (or the https://gatsbyjs.org/starters/gatsbyjs/gatsby-starter-blog/ )

Reply:
mhh - i see README is not shared - i will remove the title

Reply:
removed the filename